lightning/src/ln/channel.rs

```diff
@@ -5167,11 +5172,6 @@ impl<Signer: WriteableEcdsaChannelSigner> Channel<Signer> {
 			.filter_map(|upd| if upd.blocked { None } else { Some(&upd.update) })
 	}
 
-	/// Returns true if funding_created was sent/received.
-	pub fn is_funding_initiated(&self) -> bool {
-		self.get_context().channel_state >= ChannelState::FundingSent as u32
-	}
-
```

Comment:
This needed to be moved in the same commit as it's a dependency of one of the other methods. It technically could have been done in its own commit first, but that would have been a bit overkill.